lib/ohai/plugins/linux/hostnamectl.rb

```diff
@@ -26,7 +26,7 @@
     hostnamectl_path = which("hostnamectl")
     if hostnamectl_path
       shell_out(hostnamectl_path).stdout.split("\n").each do |line|
-        key, val = line.split(": ")
+        key, val = line.split(": ", 2)
```

Comment:
It's weird to chomp the split value... @phiggins suggestion was confusing, but what I *THINK* he meant was do:
```
line.chomp.split(": ", 2)
```
Then drop the chomp from line 30.

Reply:
Yes, I made the suggestion to drop the call to chomp before the other suggestion was made. Sorry for the confusion.